lib/kokkos/BUILD.md

```diff
@@ -40,6 +40,13 @@ cmake ${srcdir} \
 ````
 which activates the OpenMP backend. All of the options controlling device backends, options, architectures, and third-party libraries (TPLs) are given below.
 
+## Platform-specific Problems
+
+### Cray
+
+* The Cray compiler wrappers do static linking by default. This seems to break the Kokkos build. You will likely need to set the environment variable `CRAYPE_LINK_TYPE=dynamic` in order to link correctly. Kokkos warns during configure if this is missing.
```

Comment:
@akohlmey Is this referring to the `dlopen` warning? If so I think that can be safely ignored.

Reply:
no idea. that is a question for the kokkos folks. i don't have an account on a cray box currently.